cpp/src/gandiva/tests/filter_test.cc

```diff
@@ -29,7 +29,12 @@ using arrow::int32;
 
 class TestFilter : public ::testing::Test {
  public:
-  void SetUp() { pool_ = arrow::default_memory_pool(); }
+  void SetUp() {
+    pool_ = arrow::default_memory_pool();
+    // Setup arrow log severity threshold to debug level.
+    arrow::util::ArrowLog::StartArrowLog("", arrow::util::ArrowLogLevel::ARROW_DEBUG);
+    //setenv("GANDIVA_CACHE_SIZE", "5120", 1);
```

Comment:
Remove this line if it is not necessary

Reply:
Removed, it is not necessary for projector itself, I just used to check some behaviors on this test suite case.